park_api/cities/Konstanz.py

```diff
@@ -9,50 +10,45 @@ def parse_html(html):
     soup = BeautifulSoup(html, "html.parser")
 
     # last update time (UTC)
-    try:
-        date_col = soup.select('p > strong')[-1].text
-        update_time = convert_date(date_col, "Stand: %d.%m.%Y - %H:%M:%S")
-    except ValueError:
-        date_col = soup.select('p > strong')[-2].text
-        update_time = convert_date(date_col, "Stand: %d.%m.%Y - %H:%M:%S")
+    # Konstanz does not support the last_updated yet. I hope they will inform me when it's added
+    # as the data seems accurate I will return the current time and date
+    now = datetime.datetime.now()
 
     data = {
-        "last_updated": update_time,
+        "last_updated": now.strftime("%d-%m-%Y %H:%M"),
```

Comment:
This generates the wrong time format. You might want to import `utc_now` from `park_api.util` (change `from park_api.util import convert_date` to `from park_api.util import convert_date, utc_now`) and then call `utc_now()` which will create the correct format and time zone string.

Reply:
in my test-environment here I did:
`    now = datetime.datetime.now()
    print("updated: ", str(now)[0:16])
    print "Current date and time using strftime:", now.strftime("%d-%m-%Y %H:%M")
`
which outputs:
`(updated:, "2019-05-16 14:55")
Current date and time using strftime: 16-05-2019 14:55
`
and the line with the strftime() is used in the py-file.

Reply:
The required format for the output is [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) in UTC which is already provided by our `utc_now` function. The tests check that the date actually follows this format and since `"2019-05-16 14:55"` does not they fail.

Reply:
done.